docker/nginx-consul-template/nginx.conf.tmpl

```diff
@@ -0,0 +1,24 @@
+events {
+    worker_connections 4096;
+}
+
+http {
+    upstream service {
+      {{- range service (printf "%s.%s" (env "DE_ENV") (env "NGINX_PROXY_SERVICE_NAME")) }}
+      server {{.Address}}:{{.Port}} max_fails=3 fail_timeout=60 weight=1 {{- if not (.Tags.Contains (key (printf "%s/%s/color" (env "DE_ENV") (env "NGINX_PROXY_SERVICE_NAME"))))}} backup{{end}}; # {{.ID}} {{.Tags}}
+      {{- else}}
+      server 127.0.0.1:65535; # Force a 502. No {{env "DE_ENV"}}.{{env "NGINX_PROXY_SERVICE_NAME"}} services were found.
```

Comment:
It'd be nice if there were a way to explicitly return a 502 instead of relying on the fact that nobody is listening to port 65535, but I don't see a way to do it in the docs.